src/ninja.cc

```diff
@@ -56,6 +56,10 @@ int MSVCHelperMain(int argc, char** argv);
 void CreateWin32MiniDump(_EXCEPTION_POINTERS* pep);
 #endif
 
+// Use namespace ninja for now until we can carve out all
```

Comment:
Honestly I'd rather there not be any symbols at all that are outside of the ninja namespace.